test/controllers/totalCount.controller.test.js

```diff
@@ -338,4 +344,286 @@ describe('TotalCount Controller', () => {
       user: editorUser,
     });
   });
+
+  describe('Get Needs Attention Count', () => {
+    const method = 'get';
+    const endpoint = '/api/v1/total-count/need-attention';
```

Comment:
I was contemplating if I should ask you to use `needs-attention` instead of the singular form. Your description on line 348 fixes that for me 😅. Lets update the endpoint to `needs-attention`, it feels more natural to read.

```suggestion
    const endpoint = '/api/v1/total-count/needs-attention';
```